avaframe/runScripts/runComparisonModules.py

```diff
@@ -109,20 +109,23 @@
                     log.info('Reference simulation: %s' % com1DFASimName)
                     break
                 else:
-                    log.error('No reference simulation found based on releaseScenario: %s and simType: %s' % (rel, simType))
+                    log.error('No reference simulation found based on releaseScenario: %s and simType: %s' %
+                        (rel, simType))
 
         simNameRef = reportDcom1DFAOrig['simName']['name']
         refDir = pathlib.Path(avaDir, 'Outputs', 'com1DFAOrig', 'peakFiles')
         if reportDcom1DFAOrig:
             com1DFASimName = reportDcom1DFAOrig['simName']['name']
             # Fetch corresponding com1DFA
             for dict in reportDictListcom1DFA:
-                if simTypeString in dict['simName']['name'] and dict['Simulation Parameters']['Release Area Scenario'] == rel:
+                if (simTypeString in dict['simName']['name'] and
+                    dict['Simulation Parameters']['Release Area Scenario'] == rel):
```

Comment:
This is an unfortunate indent, as the dict is on the same line as the stuff that really needs to be indented.